hybrid/compartment_examples/inter_comp_call/secure/README.md

```diff
@@ -0,0 +1,10 @@
+# "inter_comp_call-secure*" examples
```

Comment:
I see we have two READMEs now. What is the intent behind there being two?

Reply:
They're separate because you are expected to run the `secure` examples from the `hybrid/compartment_examples/inter_comp_call/secure` directory and the `base` example from `hybrid/compartment_examples/inter_comp_call/base` directory (the READMEs are in the two directory you're supposed the examples from). We can merge them, but the instructions would then need to specify that you need to `cd secure` or `cd base` before running `make`.

Reply:
Right, I see. I think that would be better, to have one, and specify the two example sets (`base` and `secure`), but I have no strong feelings either way.

Reply:
I merged the READMEs into a single top-level README: https://github.com/capablevms/cheri-examples/pull/76/commits/7fb7dac0311712fe0b17a8477c298912f7292051